### ISSUE TITLE
tls: failed to find any PEM data in key input

### ISSUE THREAD

**Issue by shashank2086** [electricbubble/gidevice-cli#9]
When i am trying to execute gidevice-cli in mac. It's giving me this error
tls: failed to find any PEM data in key input

**Comment by electricbubble** [comment]
🤔

Is the `iOS device` unlocked?

> This problem seems difficult for me to reproduce

**Comment by shashank2086** [comment]
yes...but the applist command works on unlocked phones also ryt?

**Comment by shashank2086** [comment]
Sorry Locked phone...My bad

**Comment by electricbubble** [comment]
[libimobiledevice/ideviceinstaller](https://github.com/libimobiledevice/ideviceinstaller) can work on unlocked phones

But in this project, the implementation of `applist` is different

> I personally hope that the `iOS device` is unlocked

**Comment by shashank2086** [comment]
okay thanks
BDW.. i am trying to achieve codesigning of app with CL but i am facing some issues...can u help?

**Comment by electricbubble** [comment]
😵 Sorry, I don’t know much about `iOS`

**Comment by shashank2086** [comment]
lol....np
